dpd/src/transceivers/mod.rs

```diff
@@ -236,7 +242,7 @@ mod mpn_test {
             transceiver: Some(transceiver),
             management_mode: ManagementMode::Manual,
         };
-        assert_eq!(qsfp_xcvr_mpn(&qsfp).unwrap(), None);
```

Comment:
Can't comment on it, but the comment on L230 is out of date with this change.

Reply:
Fixed